test/testhelpers/withDebugLogging.ts

```diff
@@ -0,0 +1,12 @@
+import { logger } from '../../src/utils';
+
+export function withDebugLogging(fn: () => any) {
```

Comment:
I like this inclusion. Definitely helps out for potential future debug statements as well.